app/models/intake_site_drop_off.rb

```diff
@@ -106,18 +106,11 @@ class IntakeSiteDropOff < ApplicationRecord
   validates :certification_level, allow_blank: true, inclusion: { in: CERTIFICATION_LEVELS }
   validates :email, allow_blank: true, 'valid_email_2/email': true
   validates :phone_number, allow_blank: true, phone: true
-  validate :has_document_bundle?
   validate :has_valid_pickup_date?
 
   has_one_attached :document_bundle
```

Comment:
it looks like we should also remove this line in order to remove the property.

- Ben & Sarah

Reply:
D'oh, yes, I meant to do that. Doing so right now.

Reply:
We've got it covered!